### ISSUE TITLE
Explain why there are no predictions for chinstrap

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/deep-learning-intro#490]
- Do we ever explain/explore why the chinstraps are not present in the the predictions in the confusion matrix? is it the specific random seed? the train/test has stratify/shuffle=True so the right steps were taken to avoid this issue... (short of a bad seed)
- It might be worth answering so that learners can see how to tackle the "black box" nature of DL/NNs. Leaving it unanswered or saying "a bad model" doesn't seem very satisfying or good practice.

**Comment by svenvanderburg** [comment]
This refers to: https://carpentries-incubator.github.io/deep-learning-intro/2-keras.html#measuring-performance

**Comment by code4yonglei** [comment]
working on it
